tasks/fine_tuning_sbert/notebooks/Fine_tuning_SBERT_with_callback.ipynb

```diff
@@ -1,3160 +1,2839 @@
-      "text/plain": [
-       "<Figure size 576x432 with 2 Axes>"
-      ]
-     },
-     "metadata": {
-      "tags": []
-     },
-     "output_type": "display_data"
+      "cell_type": "code",
+      "metadata": {
+        "id": "ijvXrBcGlShD"
+      },
+      "source": [
+        "sweep_id = wandb.sweep(sweep_config, project=\"WRI\", entity=\"ramanshsharma\")"
+      ],
+      "execution_count": null,
+      "outputs": []
     },
     {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "============================================================\n",
-      "Current Score is: 0.9629629629629629\n",
-      "Current ACC_LIST is: [0.8148148148148148, 0.9444444444444444, 0.9382716049382716, 0.9629629629629629]\n",
-      "Epoch: 3\n",
-      "Best score: 0.9629629629629629\n",
-      "============================================================\n"
-     ]
+      "cell_type": "code",
+      "metadata": {
+        "id": "cG0F27vZlShD"
+      },
+      "source": [
+        "wandb.agent(sweep_id, train, count=5)"
+      ],
+      "execution_count": null,
+      "outputs": []
     },
     {
-     "data": {
-      "application/vnd.jupyter.widget-view+json": {
-       "model_id": "39a9eab598444ebfbe7ba5478bf171a8",
-       "version_major": 2,
-       "version_minor": 0
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "IT-nPon7lShD"
       },
-      "text/plain": [
-       "HBox(children=(FloatProgress(value=0.0, description='Iteration', max=41.0, style=ProgressStyle(description_wid…"
+      "source": [
+        "## Running training function for only one run"
```

Comment:
If this is running it once, then it's not grid search anymore right? Could we rename the function please?

Reply:
Oh yeah that's true 😅 I will change the name for it